x/perpetual/migrations/v10_migration.go

```diff
@@ -31,9 +31,7 @@ func (m Migrator) V10Migration(ctx sdk.Context) error {
 		TakeProfitBorrowInterestRateMin:                params.TakeProfitBorrowInterestRateMin,
 		SwapFee:                                        params.SwapFee,
 		MaxLimitOrder:                                  params.MaxLimitOrder,
-		FundingFeeMinRate:                              sdk.NewDecWithPrec(-111, 8),
-		FundingFeeMaxRate:                              sdk.NewDecWithPrec(111, 8),
-		FundingFeeBaseRate:                             sdk.NewDecWithPrec(33, 9),
+		FixedFundingRate:                               sdk.NewDecWithPrec(30, 2), // set to 30%
```

Comment:
@amityadav0 30% per block?

Reply:
we are changing this implementation based on time similar to stablestake to keep it consistent, so I am working on update this logic, will create another PR once done